app/src/main/java/org/covidwatch/android/exposurenotification/ENStatus.kt

```diff
@@ -3,7 +3,6 @@ package org.covidwatch.android.exposurenotification
 import androidx.annotation.IntDef
 
 sealed class ENStatus {
-    object SUCCESS : ENStatus()
```

Comment:
Flagging for myself, why?

Reply:
This is still not a stable version. The problem is that error handling is not clear as EN is not stable and different sources propose different things. For instance, the reference app doesn't use `ApiException` directly, instead it uses intent resolutions and delegates error solving to Play Services. In the EN documentation though, it says to use `ApiException` to catch errors :man_shrugging: 

Success doesn't make sense in the case of our implementation because if the operation is successful we don't get an ENStatus object at all. We receive the return value of the task.